var/spack/repos/builtin/packages/py-keras/package.py

```diff
@@ -11,8 +13,15 @@ class PyKeras(PythonPackage):
     and more. Runs on Theano or TensorFlow."""
 
     homepage = "https://keras.io"
+    git = "https://github.com/keras-team/keras"
     pypi = "Keras/Keras-1.2.2.tar.gz"
-
+    version('2.7.0',
+            url='https://github.com/keras-team/keras/archive/refs/tags/v2.7.0.tar.gz',
```

Comment:
Should we add a `url_for_version` so we don't need to specify the URL for all future versions?

Reply:
done